packages/core/src/browser/messaging/ws-connection-provider.ts

```diff
@@ -34,6 +34,12 @@ export interface WebSocketOptions {
 @injectable()
 export class WebSocketConnectionProvider extends AbstractConnectionProvider<WebSocketOptions> {
 
+    protected readonly onSocketOpenedEmitter: Emitter<void> = new Emitter();
+    public onSocketOpened: Event<void> = this.onSocketOpenedEmitter.event;
+
+    protected readonly onSocketClosedEmitter: Emitter<void> = new Emitter();
+    public onSocketClosed: Event<void> = this.onSocketClosedEmitter.event;
```

Comment:
- `public` is superfluous.
 - Should be `readonly`.
 - Nameing: `onSocketDidClose`.

Same for open.

Reply:
Renamed the names for the following emitters.